src/System.Private.CoreLib/shared/System/Exception.cs

```diff
@@ -11,6 +11,8 @@ namespace System
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public partial class Exception : ISerializable
     {
+        internal protected const string InnerExceptionPrefix = " ---> ";
```

Comment:
@danmosemsft, did you mean for this to be `private protected` rather than `internal protected`?  As it stands it's exposing this all derived types.  It's thus also breaking the corefx build.